apacheconfig/lexer.py

```diff
@@ -222,7 +222,10 @@ def _lex_option(self, t):
         if t.value.endswith('\\'):
             t.lexer.multiline_newline_seen = False
             t.lexer.code_start = t.lexer.lexpos - len(t.value)
+            if "TAG" in t.type:
```

Comment:
Just a side note: we should eventually move all these literals into some global "constants" to benefit code analyzers.

Reply:
Made an issue for it at #68